common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4053,4 +4056,21 @@ object CelebornConf extends Logging {
       .doc("Kerberos keytab file path for HDFS storage connection.")
       .stringConf
       .createOptional
+
+  val DYNAMIC_CONFIG_STORE_BACKEND: ConfigEntry[String] =
```

Comment:
Adds the below comments before this config to provider better readability.
```
  // //////////////////////////////////////////////////////
  //              Dynamic Config                //
  // //////////////////////////////////////////////////////
```

Reply:
As some exist confs would also be changed to dynamic, better not add this section, otherwise those confs need move to here.